healthcheck/routes.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
-const checks = require('./checks'),
-      outputs = require('./outputs'),
-      versionFile = require('./versionFile');
+const checks = require('./checks')
+const outputs = require('./outputs')
```

Comment:
can you separate these formatting changes to a different PR please

Reply:
done.